pages/support/index.vue

```diff
@@ -9,7 +9,7 @@ import { MANAGEMENT } from '@/config/types';
 import { getVendor, setBrand } from '@/config/private-label';
 import { SETTING } from '@/config/settings';
 
-const KEY_REGX = /^[0-9a-fA-F]{16}$/;
+const KEY_REGEX = /^[0-9a-fA-F]{16}$/;
```

Comment:
I'm used to variations, I'm not used to this one, but a colleague claimed familiarity w/ it.

Reply:
`REGEX` is definitely more common in my experience.